.github/ci_templates/e2e_tests.yml

```diff
@@ -184,6 +210,15 @@ check_e2e_labels:
 {!{- if coll.Has $ctx "manualRun" }!}
   if: needs.check_e2e_labels.outputs.run_{!{ $ctx.cri }!}_{!{ $ctx.kubernetesVersionSlug }!} == 'true'
 {!{- end }!}
+  outputs:
+    ssh_master_connection_string: ${{ steps.check_stay_failed_cluster.outputs.ssh_master_connection_string }}
```

Comment:
There are 3 names for one thing:
- `ssh_master_connection_string`
- `ssh_connection_string`
- `MASTER_CONNECTION_STRING`